docs/api/cli-options.md

```diff
@@ -219,10 +309,13 @@ The `framework-filter` argument is optional and can filter the list of available
 
 Options include:
 
-| Option                      | Description                                                                                          |
-| --------------------------- | ---------------------------------------------------------------------------------------------------- |
-| `-o`, `--output [dir-name]` | Configures the location of the sandbox project<br/>`storybook sandbox --output /my-sandbox-project`  |
-| `--no-init`                 | Generates a sandbox project without without initializing Storybook<br/>`storybook sandbox --no-init` |
+| Option                      | Description                                                                                                                                               |
+| --------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `-o`, `--output [dir-name]` | Configures the location of the sandbox project<br/>`storybook sandbox --output /my-sandbox-project`                                                       |
+| `--no-init`                 | Generates a sandbox project without without initializing Storybook<br/>`storybook sandbox --no-init`                                                      |
+| `--debug`                   | Outputs more logs in the CLI to assist debugging<br/>`storybook sandbox --debug`                                                                          |
+| `--disable-telemetry`       | Disables Storybook's telemetry. Learn more about it [here](../configure/telemetry.md)<br/>`storybook sandbox --disable-telemetry`                         |
+| `--enable-crash-reports`    | Enable sending crash reprots to Storybook's telemetry. Learn more about it [here](../configure/telemetry.md).<br/>`storybook sandbox --disable-telemetry` |
```

Comment:
The same applies here, as already mentioned for the other commands, when you're able adjust the links and wording to match the `dev` command.